security/cmd/node_agent_k8s/main.go

```diff
@@ -58,10 +58,6 @@ var (
 func init() {
 	RootCmd.PersistentFlags().StringVar(&serverOptions.UDSPath, "sdsUdsPath",
 		"/tmp/sdsuds.sock", "Unix domain socket through which SDS server communicates with proxies")
-	RootCmd.PersistentFlags().StringVar(&serverOptions.CertFile, "sdsCertFile",
```

Comment:
You should leave flags but make them do nothing, in case someone actually used these flags already.

Reply:
This is a separate branch, i think it should be okay.

Reply:
yes, it's still under development so should be fine.